### PR TITLE
fix(ci): updated version of phpunit and fixed workflow issues

### PR DESCRIPTION
**Pull request by Dobmod**
Replaced the old Travis CI badge with GitHub Actions CI badge in README.md, and all unit tests in the workflow were all green, but I am unable to test the issues that occurred on the 'release'.😁

### CODE REVIEW

**Review by leeqvip — CHANGES_REQUESTED** [review]
Please update the node version for github actions

File: composer.json [hunk 1]
```diff
@@ -42,5 +42,10 @@
             "type": "composer",
             "url": "https://asset-packagist.org"
         }
-    ]
-}
\ No newline at end of file
+    ],
+    "config": {
+        "allow-plugins": {
+            "yiisoft/yii2-composer": true
```
Comment (leeqvip) [inline composer.json:48]: Is this necessary ?

Comment (Dobmod) [reply composer.json:48]: Yeah, I have tested it. 
If not, during the 'composer install' process, a confirmation option about `yii2-composer` installation appears,and the workflow will be red. 😯

**Review by leeqvip — APPROVED** [review]

**Pull request merged**